pypika/queries.py

```diff
@@ -442,6 +444,18 @@ def having(self, criterion):
         else:
             self._havings = criterion
 
+    @builder
+    def returning(self, *terms):
```

Comment:
This code looks very similar to the `select` function. Would it be possible to consolidate the similarities between the two?